frontend/integration-tests/tests/kubevirt/vm.actions.scenario.ts

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable no-undef */
+
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '../../protractor.conf';
+import { resourceRowsPresent, filterForName, isLoaded } from '../../views/crud.view';
+import * as vmActionsView from '../../views/kubevirt/vm.actions.view';
+
+describe('Test VM actions', () => {
+  const leakedResources = new Set<string>();
+  const testVM = {
+    apiVersion: 'kubevirt.io/v1alpha2',
+    kind: 'VirtualMachine',
+    metadata: {
+      name: testName,
+      namespace: testName,
+    },
+    spec: {
+      running: false,
+      template: {
+        spec: {
+          domain: {
+            cpu: {
+              cores: 1,
+            },
+            devices: {
+              disks: [
+                {
+                  bootOrder: 1,
+                  disk: {
+                    bus: 'virtio',
+                  },
+                  name: 'rootdisk',
+                  volumeName: 'rootdisk',
+                },
+              ],
+              interfaces: [
+                {
+                  bridge: {},
+                  name: 'eth0',
+                },
+              ],
+            },
+            resources: {
+              requests: {
+                memory: '2G',
+              },
+            },
+          },
+          networks: [
+            {
+              name: 'eth0',
+              pod: {},
+            },
+          ],
+          volumes: [
+            {
+              containerDisk: {
+                image: 'kubevirt/cirros-registry-disk-demo:latest',
+              },
+              name: 'rootdisk',
+            },
+          ],
+        },
+      },
+    },
+  };
+
+  afterAll(async() => {
```

Comment:
This block of code is already present in `vm.wizard.scenario.ts`. Could you please extract it into some util file?

Reply:
Done